core/src/components/card/test/basic/card.e2e.ts

```diff
@@ -155,5 +155,12 @@ test.describe('card: basic', () => {
       const card = page.locator('ion-card');
       await expect(card).toHaveScreenshot(`card-no-content-or-header-${page.getSnapshotSettings()}.png`);
     });
+    test('should not have visual regressions with form elements inside card', async ({ page }) => {
+      await page.goto(`/src/components/card/test/form`);
+
+      await page.setIonViewport();
+
+      await expect(page).toHaveScreenshot(`card-form-${page.getSnapshotSettings()}.png`);
+    });
```

Comment:
Can we take a screenshot of just the card instead of the entire page? Also, this test should probably be split into input, textarea, and select tests in the respective component directories.

Reply:
Also we don't need Mode or RTL tests here, so we can skip them.